src/components/CustomChart/CustomChartList.tsx

```diff
@@ -151,21 +135,119 @@ export default function CustomChartList() {
         )
     }
 
+    const additionalRegistryTitleTippyContent = () => {
+        return <p className="p-12 fs-13 fw-4 lh-20">{CUSTOM_CHART_TITLE_DESCRIPTION_CONTENT.additionalParagraphText}</p>
+    }
+
+    const handleCustomChartDownload = async(e: any) => {
+        const chartRefId = e.currentTarget.dataset.versionid
+        const chartVersion = e.currentTarget.dataset.version
+        const chartName = e.currentTarget.dataset.name
+        try {
+            setDownloadInProgress(chartName)
+            const a = document.createElement('a')
+            a.href = `${Host}/${Routes.DOWNLOAD_CUSTOM_CHART}/${chartRefId}`
```

Comment:
We discussed to change this logic. right?

Reply:
Had discussed this with Utkarsh and he suggested that we can keep it this way